Source/cmake/OptionsJSCOnly.cmake

```diff
@@ -17,6 +17,7 @@ set(PROJECT_VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_
 
 WEBKIT_OPTION_BEGIN()
 WEBKIT_OPTION_DEFINE(ENABLE_STATIC_JSC "Whether to build JavaScriptCore as a static library." PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_LIBBACKTRACE "Whether to enable usage of libbacktrace." PUBLIC OFF)
```

Comment:
Out of curiosity, is libbacktrace available by default in macOS SDKs? (I don't think so, but maybe you know better).

Reply:
Yeah, this patch seems to have some issues. I thought turning this off by default wouldn't break anything, but it is still failing EWS. I'll look into it